swap_meet/vendor.py

```diff
@@ -1,2 +1,75 @@
+#Wave 1
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item 
+    
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+
+        return False
```

Comment:
So it should be: 

        if item not in self.inventory:
            return False
        
        self.inventory.remove(item)
        return item
        
 so that we avoids unnecessary operations in case the item is not in the inventory.